perllib/FixMyStreet/Cobrand/Rutland.pm

```diff
@@ -40,6 +40,19 @@ sub open311_extra_data_include {
     ];
 }
 
+sub open311_contact_meta_override {
+    my ($self, $service, $contact, $meta) = @_;
+
+    my ($hint) = grep { $_->{code} eq 'hint' } @$meta;
+    my ($group_hint) = grep { $_->{code} eq 'group_hint' } @$meta;
```

Comment:
Do you want to delete these rows from $meta, so that they're not also extra attributes that don't do anything?